### ISSUE TITLE
video stream+app

### ISSUE THREAD

**Issue by manang** [hardening/Cave#1]
Hi,
i'm using your phantom3-video application. the problem is that I can stream the video only if the application on the tablet is closed.
is it possible to use the streaming in linux OS and contemporary the application on the tablet?
thanks

**Comment by hardening** [comment]
Hi,
not that I know, the problem is that the camera speaks to only 1 client in udt, so it's either the tablet app or the desktop. The best would be that I would have enough time to work on a desktop app that could compete with the tablet app, at least for basic functions.

**Comment by manang** [comment]
Ok, a question to avoid this problem in this my first approach with dji:
Do you know if I can create an android application to take the photos or videos triggered by the pc? I mean an application that permits to have the dji go application running and the custom application running as service?

do you know if libudt is available for arm (raspberry pi)?

**Comment by hardening** [comment]
Yes libudt is available on almost all platforms, and especially on ARM as I guess that it's exactly what is running on the camera (which has an ARM CPU IIRC). But again the limitation is in the camera itself that will honour only one client at the same time.
